Nested helpers don't overwrite helper reference (#1546)
spacetelescope/jdaviz: CHANGES.rst

  and visibility states are now always adopted from the existing entry that would be overwritten.
  [#1538]

> - The top-level config will now always be returned via ``app._jdaviz_helper`` and any associated

Reviewer: Change log is for the users and I don't think users will understand what this means for them. It should only mention the immediate bug that this fixes; i.e., an entry under bug fixes for Cubeviz that says calling cubeviz.specviz will no longer overwrites its viewer.jdaviz_helper . Do you agree?

Author: Sure, I can rephrase it. Let me give this another stab

- Fixed validation message of moment number in moment map plugin. [#1536]

> - Fixed ``app._jdaviz_helper`` returning Specviz helper instead of Cubeviz helper when Specviz
>   helper is called. ``app._jdaviz_helper`` always returns Cubeviz helper [#1546]

Reviewer: Hidden attribute should be hidden from user, so I propose a little more changes to the verbiage. 😬 

```suggestion
- Fixed ``viewer.jdaviz_helper`` returning Specviz helper instead of Cubeviz helper after Specviz
  helper is called via ``Cubeviz.specviz``. Now ``viewer.jdaviz_helper`` always returns the Cubeviz helper. [#1546]
```

Author: Ok Happy to adopt. I changed it to the hidden attribute because that was the root cause, but I can see your point
